### PR TITLE
Refactor OBS_GetAvailableTimespans

### PR DESCRIPTION
**Pull request by antoniocarlon**
- Refactor `OBS_GetAvailableTimespans`
 - Updated documentation

Closes https://github.com/CartoDB/bigmetadata/issues/422

### CODE REVIEW

File: src/pg/sql/42_observatory_exploration.sql [hunk 1]
```diff
@@ -405,6 +405,13 @@ BEGIN
 END
 $$ LANGUAGE plpgsql;
 
+DROP FUNCTION IF EXISTS cdb_observatory.OBS_GetAvailableTimespans(
```
[the remark this answers is not in the document]
Comment (antoniocarlon) [reply src/pg/sql/42_observatory_exploration.sql]: You're right. Removed!

**Comment by ethervoid** [comment]
@antoniocarlon tests are failing
```
  
! CONTEXT:  PL/pgSQL function cdb_observatory.obs_getavailabletimespans(geometry,text[],text,text,text) line 14 at RETURN QUERY
! ERROR:  column "timespan_alias" does not exist
! LINE 15:            timespan_alias::TEXT,
!                     ^
 ```

**Comment by antoniocarlon** [comment]
Fixed tests for Postgres 9.5 and 9.6 (PostGIS 2.2 and 2.3), still failing on 10 (PostGIS 2.4)

**Comment by ethervoid** [comment]
Blocked until we have finished timespans PR

**Pull request merged**